R/coh_data_wrangling.R

```diff
@@ -371,16 +387,11 @@ get_time_to_event <- function(data, outcome_date_col,
     )
   }
 
-  # convert strings to dates
-  start_cohort <- as.Date(start_cohort)
-  end_cohort <- as.Date(end_cohort)
-
-  # convert outcome date to Date type
-  data[[outcome_date_col]] <- as.Date(data[[outcome_date_col]])
-
   if (start_from_immunization) {
     # convert date columns to Date type
-    data[[immunization_date_col]] <- as.Date(data[[immunization_date_col]])
+    checkmate::assert_date(
+      data[[immunization_date_col]]
+    )
```

Comment:
Same as above - confirm whether missing values are allowed here.

Reply:
Allowed. In this case not having immunization date means unvaccinated.